iFixit/react-commerce
# Add lifetime guarantee banner

This PR adds the collection page lifetime guarantee banner.


## QA

1. Visit the Vercel PR preview
2. Visit any collection page (other than `/parts` and `/tools`)
3. Verify that the lifetime guarantee banner is displayed and looks like the mockup (see below for reference):

    <img width="727" alt="Screen Shot 2021-05-28 at 17 59 08" src="https://user-images.githubusercontent.com/4640135/120011289-6995c500-bfde-11eb-9253-9b3c03951152.png">

4. Verify that the banner is responsive
